app.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const Sequelize = require('sequelize');
+
+const app = express();
+const connection = new Sequelize({
```

Comment:
Please move this connection into `app/models/index.js`. And then I'll show you how you can load all the models automatically.